runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -220,6 +222,8 @@ public ClientUpdateCheckMode convert(String v)
 		final RuntimeMXBean rb = ManagementFactory.getRuntimeMXBean();
 		final long uptime = rb.getUptime();
 		log.info("Client initialization took {}ms. Uptime: {}ms", end - start, uptime);
+
```

Comment:
revert changes to this file

Reply:
ok reverting it now, i added something when testing out git commands and forgot what I changed